lib/plugins.ts

```diff
@@ -1,7 +1,9 @@
-var webdriver = require('selenium-webdriver');
-import {Logger} from './logger';
-import * as Q from 'q';
+import * as q from 'q';
```

Comment:
However, in the [docs](https://github.com/kriskowal/q/wiki/API-Reference) for that module, the uppercase Q is used...